easybuild/easyconfigs/m/Mesa/Mesa-10.5.5-intel-2015a-Python-2.7.10.eb

```diff
@@ -69,6 +68,6 @@ sanity_check_paths = {
     'dirs': []
 }
 
-maxparallel = 1
+#maxparallel = 1
```

Comment:
It was a really slow build and I wanted to see if it worked in parallel...and it did so I left it commented out.